### PR TITLE
Components/radio

### PR DESCRIPTION
**Pull request by samcyn**
Related to #57 
Added radio button components and its features.
 
**Available props**
- disabled,
-  initialValue,
-  label,
-  labelStyle,
-  onChange,
-  styles,
-  theme,

### CODE REVIEW

File: src/Radio.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  // O N P R E S S - H A N D L E R
+  radioPressHandler() {
+    this.setState({ checked: !this.state.checked }, () => this.props.onChange(this.state.checked));
+  }
+
+  render() {
+    const { props, state } = this;
+    const {
+      styles,
+      disabled,
```
Comment (palingheorghe) [inline src/Radio.js]: the `disabled` prop doesn't change the color of the radio circle to gray, this should change the color as well, not only the text

File: src/Radio.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      flexDirection,
+      radioContainerStyle,
+      radioOutterPartStyle,
```
Comment (palingheorghe) [inline src/Radio.js]: `radioOutterPartStyle` -> `radioOuterStyle`

File: src/Radio.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    this.setState({ checked: !this.state.checked }, () => this.props.onChange(this.state.checked));
+  }
+
+  render() {
+    const { props, state } = this;
+    const {
+      styles,
+      disabled,
+      flexDirection,
+      radioContainerStyle,
```
Comment (palingheorghe) [inline src/Radio.js]: `radioContainerStyle` -> `containerStyle`

File: src/Radio.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+      radioOutterPartStyle,
+      radioInnerPartStyle,
```
Comment (palingheorghe) [inline src/Radio.js]: `radioInnerPartStyle` -> `radioInnerStyle`

File: src/Radio.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    } = props;
```
Comment (palingheorghe) [inline src/Radio.js]: add a prop called `color` which accepts all the colors Galio has (as a string) and any type of color the user decides to put in in order to change the radio ring faster

File: src/Radio.js (hunk 1 above)
Comment (samcyn) [reply src/Radio.js]: oh that's true and noted. Thanks for the review

File: src/Radio.js (hunk 5 above)
Comment (samcyn) [reply src/Radio.js]: and another great one here. Thanks for your reviews. Changes will make up soon. Reards

**Comment by samcyn** [comment]
Hello @palingheorghe,  color, containerStyle, radioOuterStyle, and radioInnerStyle props are added. Hit me up with your reviews. Thanks.

**Comment by palingheorghe** [comment]
@samcyn this is perfect! Thank you for your PR, we can now safely update Galio to the next version! 

Thank you very much for your help!
Alin 🧜🏽‍♂️🔥

**Comment by ankitv89** [comment]
Is there any way we can dynamically change state of radio buttons?

**Comment by palingheorghe** [comment]
@ankitv89 at the moment there's no way of changing the radio buttons. You could always rise up a PR and help us out with this important feature <3

**Comment by ankitv89** [comment]
Hi @palingheorghe , I can help with that.

**Comment by palingheorghe** [comment]
looking forward to your PR @ankitv89

**Pull request merged**